{"name":"MongoDB.prevError","purpose":"Checks for the last error thrown during a database operation","manualid":"mongodb.preverror","version":false,"params":[],"return":{"type":"array","description":"Returns the error and the number of operations ago it occurred."},"errors":null,"notes":[],"changelog":[],"seealso":[]}